sdk/database/helper/connutil/sql.go

```diff
@@ -12,15 +12,36 @@ import (
 	"sync"
 	"time"
 
+	"github.com/hashicorp/go-uuid"
+
 	"github.com/hashicorp/errwrap"
 	"github.com/hashicorp/go-secure-stdlib/parseutil"
+	"github.com/mitchellh/mapstructure"
+
 	"github.com/hashicorp/vault/sdk/database/dbplugin"
 	"github.com/hashicorp/vault/sdk/database/helper/dbutil"
-	"github.com/mitchellh/mapstructure"
+)
+
+const (
+	AuthTypeIAM = "gcp_iam"
```

Comment:
Do we expect more IAM auth types in the future? Should this be AuthTypeIAMGCP?

Reply:
updated to AuthTypeGCPIAM (Austin commented on the same thing)